jvector-base/src/main/java/io/github/jbellis/jvector/quantization/NVQuantization.java

```diff
@@ -355,6 +360,37 @@ public String toString() {
         return String.format("NVQuantization(sub-vectors=%d)", subvectorSizesAndOffsets.length);
     }
 
+    @Override
+    public double reconstructionError(VectorFloat<?> vector) {
+        var encodedVector = QuantizedVector.createEmpty(subvectorSizesAndOffsets, bitsPerDimension);
```

Comment:
nit: just a standard practice to mark as `final` variables that are not to be re-assigned (same convention as in c++ with `const`)